### DISCUSSION TITLE
Asset delete api

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Knowlg/Community#187] [General Discussions]
Hello Team,

As we're currently on release 5.0, I'm in need of assistance regarding the asset delete API. While I've managed to obtain the cURL command for it, unfortunately, the corresponding code for this API isn't documented. Could someone kindly specify in which release the asset delete code will be available?

Below is the cURL command for reference:

`curl --location --request POST '{{host}}/api/asset/v1/delete/{{assets_id}}' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer ' \
--header 'x-authenticated-user-token:  \
--header 'X-Channel-ID: '`

Additionally, I've attached a screenshot showcasing the API list as declared in the config file for further context.
![Screenshot from 2024-02-08 09-52-24](https://github.com/Sunbird-Knowlg/Community/assets/72926048/dc4daddc-baaf-4026-a690-4ad263b4e91e)

Your guidance on this matter would be greatly appreciated. Thank you in advance for your help!

**Comment by pallakartheekreddy** [comment]
@siddhishinde0723 At present, our Asset APIs do not include an `asset deletion endpoint`. It would be greatly appreciated if you could implement and contribute this feature back to the project. If you're interested in working on it, feel free to use this thread to discuss the design details.

I just want to confirm where you found the endpoint mentioned below.
`{{host}}/api/asset/v1/delete/{{assets_id}}`

@mohitga

**Reply by siddhishinde0723** [reply]
@pallakartheekreddy the curl i recieved from [vinukumar-vs](https://github.com/vinukumar-vs) 
Please refer the discussions  for the same [link](https://github.com/orgs/Sunbird-Ed/discussions/661)
